components/date-picker/style/panel.less

```diff
@@ -538,11 +538,7 @@
   }
 
   // ====================== Date Panel ======================
-  &-date-panel {
-    .@{picker-prefix-cls}-body {
-      padding: @padding-xs @padding-sm;
-    }
```

Comment:
Could you also undo the changes here? In general, we will not change the styles here, because it's from ant-design.

Reply:
This is to support week numbers. This style is still applied, just not when week numbers are shown. Without this change the week numbers will cause the dates to be cut off.

Reply:
If this is necessary, add the styles on patch.less while keep the origin file.